internal/actions/pr.go

```diff
@@ -756,6 +757,12 @@ func readLineUntil(b *bytes.Buffer, line string) error {
 	for {
 		l, err := b.ReadString('\n')
 		if err != nil {
+			// If ReadString encounters an error before finding a delimiter, it returns the data read before the error and the error itself (often io.EOF).
+			// https://pkg.go.dev/bytes#Buffer.ReadString
+			if err == io.EOF && l+"\n" == line {
```

Comment:
If a PR already exists, the trailing newline gets removed here https://github.com/aviator-co/av/blob/master/internal/actions/pr.go#L344 and it no longer matches with `PRMetadataCommentEnd` https://github.com/aviator-co/av/blob/master/internal/actions/pr.go#L675. What would be a good way to fix this?

Reply:
@draftcode Could you give me advice when you have time?

Reply:
Sorry, I tried to adopt another approach. Please ignore my comment